index.html

```diff
@@ -0,0 +1,93 @@
+
+
+ <br><label for="dropdown" >Which option best describes your current role?</label>  
+<select id="dropdown">
+    <option value ="your-skills" disabled> select current role </option>
+    <option value="student">Student</option>
+    <option value="full-time-job">Full time job</option>
+    <option value="full-time-learner">Full time learner</option>
+    <option value="prefer-not-to-say">Prefer not to say</option>
+    <option value="other">Other</option>
+
+</select>
+
+ <legend>Would you recommend freeCodeCamp to a friend?</legend>
+ <input type="radio" checked name="recommended" id ="definate" value="definety"><label for="definetly">Definetly</label> <br>
+ <input type="radio" name="recommended" id ="maybe" value="maybe"><label for="maybe">Maybe</label> <br>
+ <input type="radio" name="recommended" id ="not-sure" value="not-sure"><label for="not-sure">Not Sure</label> <br>
+    <label for=""></label>
+
+ <br><label for="dropdown" >What is your favorite feature of freeCodeCamp?</label> 
+ <select id ="second-dropdown">
+     <option value="select-an-option" disabled> Select an option</option> 
+      <option  value="challenge">Challenge</option>
+      <option value="projects">Projects</option>
+      <option value="community">Community</option>
+      <option value="open-source">Open source </option>
+ </select>
+
+
+
+
+<p>What would you like to see improved? (Check all that apply)</p>
+<input type="checkbox" id="front-end" name="font-end" value="font-end">
+<label for="fonte-end"> Front-end Projects</label> <br>
+<input type="checkbox" id="back-end" name="back-end" value="video-quality">
+<label for="Back-end-Projects"> Back end project</label> <br>
+<input type="checkbox" name="data-visulization" id="data-visulization" value="data-viualization">
+  <label for="data-visulization" >Data visualization</label> <br>
+<input type="checkbox" name="challenge" id="challenge" value="challenge">
+<label for="challenge"> Challenges</label> <br>
+<input type="checkbox" id ="open-source-community"  value="open-source communtiy">
+<label for ="open-source-communtiy" > Open source community  </label><br>
+    <input type="checkbox" id ="gitter-help-rooms"  value="Gitter-help-rooms">
+    <label for="Gitter-help-rooms" > Gitter help rooms </label><br>
+    <input type="checkbox"  id="videos" value="videos">
+    <label for="videos">Videos</label><br>
+    <input type="checkbox" id="city-meetups" value="city-meetups">
+    <label for="city-meetups">City meetups</label> <br>
+    <input type="checkbox" id ="wiki" value="wiki">
+    <label for="wiki">Wiki</label><br>
+    <input type="checkbox"  id ="forum" value="forum">
+    <label for="forum">Forum</label> <br>
+    <input type="checkbox" id ="additional-courses" value="additional-courses">
+    <label for="additional-courses"> Additional Courses</label>
+ 
+
+
+
+<p> Any comments or suggestions?</p>
+<textarea row="4" cols="50" id="comment" p>
```

Comment:
- [ ] Please use a placeholder type instead of a `<p>` this will make your project looks more  professional